tests/sentry/integrations/slack/test_utils.py

```diff
@@ -103,3 +103,193 @@ def test_rate_limiting(self):
         )
         with pytest.raises(ApiRateLimitedError):
             get_channel_id(self.organization, self.integration, "@user")
+
+
+class GetChannelIdFasterTest(TestCase):
+    @with_feature("organizations:slack-use-new-lookup")
```

Comment:
The `setUp` and `tearDown` methods don't need the `@with_feature` decorator, it's just used when you want to force the `get_channel_id` code path to use the new stuff.